sdk/testing/src/main/java/com/mesosphere/sdk/testing/BaseServiceSpecTest.java

```diff
@@ -39,7 +39,12 @@ protected void testYaml(String fileName) throws Exception {
     }
 
     protected void deserializeServiceSpec(String fileName) throws Exception {
-        File file = new File(getClass().getClassLoader().getResource(fileName).getFile());
+        File file;
+        try {
+            file = new File(getClass().getClassLoader().getResource(fileName).getFile());
+        } catch (NullPointerException e) {
```

Comment:
I left this in so that the user would get a more helpful message than a NullPointerException if they forget to link a file to the resources folder, or have some other trivial problem. I can remove if you disagree.